features/credential-management.yml.dist

```diff
@@ -0,0 +1,71 @@
+# Generated from: credential-management.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2020-01-15
+  baseline_high_date: 2022-07-15
+  support:
+    chrome: "60"
+    chrome_android: "60"
+    edge: "79"
```

Comment:
This hinges on `navigator.credentials.store`, and I've confirmed that is missing from Edge 18, BCD is correct. This does affect the Baseline low date, so I'd like a second opinion from a reviewer on whether this is right, or if we think that `store()` is a nice-to-have. One data point is that caniuse says Edge 79.

Reply:
I had a look around and `store()` appears in lots of examples for this API (e.g., the first few blog posts I found via Google). Whether it's strictly essential to the feature or not, it does appear to be part of the popular understanding of what this thing is. Let's keep it.